### PR TITLE
renamed package

### LINKED ISSUE #8: Rename package
The package name should be aligned to the new name of `rdf-native-loader`, as discussed in [this issue](https://github.com/zazuko/rdf-native-loader/issues/4).

I would suggest: `rdf-loader-code`

### PR DESCRIPTION
**Pull request by bergos**
fixes #8

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ to be executed in their programs.
 Currently must be installed from git
 
 ```
-npm i --save https://github.com/zazuko/rdf-native-loader-code
-npm i --save https://github.com/zazuko/rdf-native-loader
+npm i --save https://github.com/zazuko/rdf-loader-code
+npm i --save https://github.com/zazuko/rdf-loaders-registry
```
Comment (tpluscode) [inline readme.md]: `rdf-loaders-registry` is not a dependency? Maybe peer dependency? @vhf

File: readme.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ to be executed in their programs.
 Currently must be installed from git
 
 ```
-npm i --save https://github.com/zazuko/rdf-native-loader-code
-npm i --save https://github.com/zazuko/rdf-native-loader
+npm i --save https://github.com/zazuko/rdf-loader-code
```
Comment (tpluscode) [inline readme.md]: Maybe this can already be changed to `npm I --save rdf-loader-code rdf-loaders-registry`?

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "rdf-native-loader-code",
+  "name": "rdf-loader-code",
```
Comment (tpluscode) [inline package.json]: `@zazuko` prefix?

File: readme.md (hunk 1 above)
Comment (vhf) [reply readme.md]: If rdf-loader-code cannot be used without rdf-loaders-registry but doesn't `require` it, peer dependency is the right choice IMO :)

**Comment by bergos** [comment]
@tpluscode @vhf which name should I use for the Github repo?

File: readme.md (hunk 1 above)
Comment (bergos) [reply readme.md]: It's possible to use a loader without the registry and it makes sense for known structures to enforce a specific result. So I don't think a peer dependency should be added.

File: readme.md (hunk 2 above)
Comment (bergos) [reply readme.md]: See comment about peer dependency.

Comment (tpluscode) [reply readme.md]: These comments are note related

I meant to change the git dependencies into package names

Comment (bergos) [reply readme.md]: Ups, sorry, didn't see the follow-up line in the conversation tab, so it looked like it's about installing 1 vs 2 packages. Yes, makes sense. Will fix it.

Comment (tpluscode) [reply readme.md]: While you're at it, the examples are also outdated...

**Pull request merged**